data/ir-ads

```diff
@@ -0,0 +1,361 @@
+# List of ads and tracking company in Iran.
+# Thanks to PersianBlocker project
+# https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlockerHosts.txt
+
+#-------------------------v Persian Ads/Trackers/Annoyances Filters v-----------------------#
+# >> A <<
+analyt.ir
+
+scripts-ads.s3.ir-thr-at1.arvanstorage.com
+error-tracking.arvancloud.com
+
+adexo.ir
+adskav.com
+advn.ir
+
+amaroid.net
+lnk.amaroid.net
+
+analytics.aasaam.com
+analytics-2.aasaam.com
+analytics-3.aasaam.com
+
+adwised.com
+adwisedfs.com
+adwised.azureedge.net
+# TODO: Remove adwised.azureedge.net, but navatv.com and baran-music.com still links to it
+
+adivery.com
+adnegah.net
+adpulse.ir
+adsima.net
+adtrace.io
+affili.ir
+amarfa.ir
+ayyaar.ir
+# ^ Used by irna.ir
+
+adro.co
+affstat.adro.co
+affstat.digikala.com
+affiliate.digikala.com
+
+# >> B <<
+behinava.com
+radar.bayan.ir
+backority.ir
+
+# >> C <<
+cayot.ir
+# Example: tamasha.com/v/DDlMB
+
+chavosh.org
+ck.chavosh.org
+
+clickaval.com
+clickyab.com
+chabok.io
+
+congoro.com
+congoro.ir
+
+# >> D <<
+24d.ir
+l.24d.ir
```

Comment:
The domain `24d.ir` will match the domain `*.24d.ir`, so you don't need to add `l.24d.ir`. Please check the domain names submitted by you. And fix this kind of situation.